core/server/src/main/webapp/browse-body.jsp

```diff
@@ -16,6 +16,33 @@
 <%@ page import="static java.net.URLEncoder.encode" %>
 
 <jsp:include page="header-scripts.jsp" />
+
+<script>
```

Comment:
If possible, add this script to the parent jsp page and place above the closing `</body>` tag. If not possible because of the need to set `currentDir` then it's ok.